### ISSUE TITLE
Create some model DOCX templates with common template tasks, like signature block, certificate of service, case caption, etc.

### ISSUE THREAD

**Issue by plocket** [SuffolkLITLab/AssemblyLine-docs#100]
On the card:

> - [ ] Avoid typing in all caps. Use the all caps font setting instead.
> - [ ] Use standardized signature blocks [what does this mean? link?]
> - [ ] Use standardized address block [link?]
> - [ ] Use standardized certificate of service [link?]

**Comment by plocket** [comment]
This one is not one I can complete alone as I don't know what the items in the DOCX checklist all mean.

**Comment by plocket** [comment]
Feedback:
> The checklist could say "look at existing certificates of service to avoid duplicating effort"

**Comment by plocket** [comment]
What I've come up with so far (that will need review):

[example_docx_checklist_items.docx](https://github.com/SuffolkLITLab/docassemble-AssemblyLine-documentation/files/5997235/example_docx_checklist_items.docx)

**Comment by nonprofittechy** [comment]
The main point of all three checklist items is to have people working on DOCX forms to copy in standard formatted examples.

The same things come up again and again. E.g., you want to use the .address.block() method and have it in a table so the lines are all indented correctly.

You want to hide the signature on the preview version of the document.

The certificate of service can be done a million different ways, but there's no reason to create one from scratch. It's more sensible to use the same variable names, date formats, etc. as our best example form and copy the appropriate logic from the best example form.

Each of those three points could be handled by an inserted DOCX sub-template or a method on the ALIndividual class. Next-best would be at least a model DOCX file that they can copy code from. Without doing that work and to get the checklists complete, I suggest re-phrasing the checklist items to something that just tells the person to not reinvent the wheel for any of those three items and to look at existing forms.

**Comment by nonprofittechy** [comment]
> 
> 
> What I've come up with so far (that will need review):
> 
> [example_docx_checklist_items.docx](https://github.com/SuffolkLITLab/docassemble-AssemblyLine-documentation/files/5997235/example_docx_checklist_items.docx)

What would be helpful for a user probably is to have a working DOCX table that makes the address block indented about 5 inches. Same with signature. The Jinja is helpful too, but I think that would make this better.

I don't know if any one document has a perfect model for this yet. For later reference--this is also not perfect yet--take a look at how I did this in MADE here: https://github.com/GBLS/docassemble-MAEvictionDefense/blob/master/docassemble/MAEvictionDefense/data/templates/include_SignatureBlock.docx . One big thing wrong with that as the model is that the certificate of service is something that should be in its own file and a bit more distinctive for most of our forms.

This style is one that I learned from Capstone, so a lot of HotDocs developers will also be familiar with the idea of small sub-templates for this kind of thing.

**Comment by plocket** [comment]
For now, edited checklist items.

**Comment by samglover** [comment]
I don't think this is necessary any more given the [DOCX page](https://assemblyline.suffolklitlab.org/docs/docassemble_intro/working-with-docx) and the new DOCX labeler tool.
